nailgun/entities.py

```diff
@@ -6814,8 +6814,12 @@ def __init__(self, server_config=None, **kwargs):
                 'remote_execution_proxy':
                     entity_fields.OneToManyField(SmartProxy),
                 'subnet_parameters_attributes': entity_fields.ListField(),
+                'template': entity_fields.OneToOneField(SmartProxy),
```

Comment:
btw, mind removing this check (`if _get_version(server_config) >= Version('6.1')`) completely? It's a leftover from times when we had 1 branch for all satellite versions, now it doesn't make sense and 2 separate dicts aren't that readable.

Reply:
Done, however Travis checks don't like that, looking into it

Reply:
looks like the [test_subnet_fields](https://github.com/SatelliteQE/nailgun/blob/master/tests/test_entities.py#L3626) is obsoleted, so I'm removing it